### DISCUSSION TITLE
Options for create admin users from command line

### DISCUSSION THREAD

**Discussion by ramir1** [moonshine-software/moonshine#440] [General]
`
12  -    protected $signature = 'moonshine:user';
12 +    protected $signature = 'moonshine:user {--username=} {--name=} {--password=}';

18 -     $username = $this->ask(
18 +     $username = $this->option('username') ?: $this->ask(
          
 24 -     $name = $this->ask('Name');
 24 +    $name = $this->option('name') ?: $this->ask('Name');

 25  -   $password = $this->secret('Password');
 25  +   $password = $this->option('password') ?: $this->secret('Password');
`

**Comment by lee-to** [comment]
Почему не сделаете pr ?

**Reply by ramir1** [reply]
Не знаю, как делать )) Что-то "поклацал", не знаю, может получилось )
